### ISSUE TITLE
Studying together & Study space sharing

### ISSUE THREAD

**Issue by DrumsnChocolate** [csvalpha/amber-ui#781]
A number of people would like to be able to indicate to other people where they are studying, and vice versa want to see where others are studying, with the idea that this will make it easier to study together

**Comment by lodewiges** [comment]
is this still relevant or would this be solved with #845?

**Comment by lodewiges** [comment]
you are correct but the tweede kamer in central place to do this for alpha maybe good to poll if people would use this feature

**Comment by DrumsnChocolate** [comment]
I would not wake sleeping dogs. Don't do polls about a feature if you do not intend to build it.

**Comment by DrumsnChocolate** [comment]
on the other hand, if you want to build a new feature, go ahead. But I suspect this one may be quite a bit of work
